src/main/kotlin/ru/hse/dynamomock/service/AWSDynamoDBMockService.kt

```diff
@@ -208,13 +215,96 @@ class AWSDynamoDBMockService(private val storage: DataStorageLayer) {
                 storage.updateItem(it)
             }
         }
-        deleteItemRequests.forEach{ storage.deleteItem(it) }
+        deleteItemRequests.forEach { storage.deleteItem(it) }
 
         return BatchWriteItemResponse.builder()
             .build()
     }
 
-    private fun getKeyFromMetadata(keyName: String, keys: Map<String, AttributeValue>, attributeDefinitions: List<AttributeDefinition>): Key {
+    fun scanItems(fileName: String, tableName: String) {
+        val allowedTypes = EnumSet.of(SS, S, N, NS, L, M, BOOL, NULL)
+        val columnsInfo = mutableListOf<Pair<String, String>>()
+
+        File(fileName).useLines { line ->
+            val firstRow = line.firstOrNull() ?: throw AWSMockCSVException("The file is empty")
+            val info = firstRow.split(";")
+            info.forEach {
+                val value = it.split("|")
+                if (value.size != 2) {
+                    throw AWSMockCSVException("Wrong value format. Use <column_name>|<type>")
+                }
+                val columnName = value[0].trim()
+                val type = value[1].trim()
+
+                if (!allowedTypes.contains(DynamoType.valueOf(type))) {
+                    throw AWSMockCSVException("Function scanItems supports only S, N, NS, SS, NULL, BOOL types right now")
```

Comment:
It's better to use String representation of allowedTypes for info, to consistency between actual tyes and this message